AutoMapper.Data.Tests/DataReaderMapping.cs

```diff
@@ -395,9 +401,8 @@ public IDataReader BuildDataReaderWithNullableField()
         public When_mapping_a_data_reader_to_a_dto_with_nullable_enum()
         {
             Mapper.Initialize(cfg => {
-                MapperRegistry.Mappers.Insert(0, new DataReaderMapper());
-
-                cfg.CreateMap<IDataReader, DtoWithSingleNullableField>();
+                cfg.Mappers.Insert(0, new DataReaderMapper());
+                cfg.AddMemberConfiguration().AddMember<DataRecordMemberConfiguration>();

```

Comment:
IF you need both calls to get DataReaderMapper to work.  Can you make this an extension method and call that?
And if so maybe update the ReadMe.md as well to reflect this.